packages/esm-patient-chart-app/src/patient-flags/patient-flags-list.component.tsx

```diff
@@ -0,0 +1,197 @@
+import React, { useMemo, useState } from 'react';
+import debounce from 'lodash-es/debounce';
+import isEmpty from 'lodash-es/isEmpty';
+import orderBy from 'lodash-es/orderBy';
+import { Toggle, ButtonSet, Button, Search, Form, Dropdown, Stack } from '@carbon/react';
+import { useLayoutType, showToast, showNotification, parseDate, formatDate } from '@openmrs/esm-framework';
+import { DefaultWorkspaceProps } from '@openmrs/esm-patient-common-lib';
+import { useTranslation } from 'react-i18next';
+
+import { useFlagsFromPatient, enablePatientFlag, disablePatientFlag } from './hooks/usePatientFlags';
+import Loader from '../loader/loader.component';
+import { getFlagType } from './utils';
+
+import styles from './patient-flags-list.scss';
+
+const PatientFlagsList: React.FC<DefaultWorkspaceProps> = ({ patientUuid, closeWorkspace }) => {
+  const { flags, isLoadingFlags, flagLoadingError, mutate } = useFlagsFromPatient(patientUuid);
+
+  const enabledFlagsUuid = flags.filter((f) => !f.voided).map((f) => f.uuid);
+
+  const [enabling, setEnabling] = useState(false);
+  const [disabling, setDisabling] = useState(false);
+
+  const [sortBy, setSortBy] = useState('A - Z');
```

Comment:
I'd suggest having an enum
enum SortingOptions {
alphabetically: "A_Z",
...
}

This will help in better checks for the strings used later in the file